python/perf-kernels/flash-attention.py

```diff
@@ -387,6 +420,19 @@ def attn_fwd(
         )
     else:
         bias_ptr = None
+
+    if USE_ALIBI != 0:
+        alibi_slopes_ptr = tl.make_block_ptr(
+            base=alibi_slopes,  # The base pointer to the parent tensor
```

Comment:
A bit nitpicky, but as we will eventually upstream this and it will be representing AMD flash attention, a minor code style suggestion would be to move these comments up to the first block pointer declaration. A reader can assume the others follow the same concept.

Reply:
The notes where mostly for me. I removed them.